apps/consultation-portal/infra/samradsgatt.ts

```diff
@@ -18,6 +19,7 @@ export const serviceSetup = (): ServiceBuilder<'consultation-portal'> =>
     .env({
       BASEPATH: '/consultation-portal',
       ENVIRONMENT: ref((h) => h.env.type),
+      API_URL: ref((h) => `http://${h.svc(consultationService)}`),
```

Comment:
This returns the URL of the consultation portal but you need the URL of the GraphQL API. 

Check out how the api service is passed into the web factory here so it can call `h.svc` on it: https://github.com/island-is/island.is/blob/cb067e59c154736fc31ad9f15ee2931b134f8a34/infra/src/uber-charts/islandis.ts#L95

You'll need to tweak that islandis.ts file and move the consultation portal variable below the api service variable.

Reply:
Done :) Thanks!